web/client/components/map/openlayers/DrawSupport.jsx

```diff
@@ -1278,13 +1278,13 @@ export default class DrawSupport extends React.Component {
     };
 
     olGeomFromType = ({type, coordinates, radius, center, projection, options}) => {
-
+        // TODO check correct number of nesting arrays of coordinates for each case
```

Comment:
with some sort of recursion by checking the depth of an array. 
not doing it  now because the fix was a regression and i just restored the old working code.
anyway passing a wrong coord per geom type will not cause error in ol, but rather the rendered absence of the feature